### DISCUSSION TITLE
Slow response on basic go microservice

### DISCUSSION THREAD

**Discussion by ghost** [vercel/vercel#5197] [Help] [answered]
The response time is around 1.1s for a simple http handler ( stated in examples ): 
```
package test

import (
	"fmt"
	"net/http"
)

func Handler(w http.ResponseWriter, r *http.Request) {
	fmt.Fprintf(w, "<h1>Hello from Go!</h1>")
}
```

This happens on both production and development, is it normal?

Thanks for reading.

**Comment by paulogdm** [comment, marked as answer]
You need to consider the region where your function is being deployed to and also the "cold start" time. It can be normal depending on a number of factors. For simple examples, the "cold boot" penalty can be around 200-800ms.

**Reply by ghost** [reply]
I see, thanks for pointing out regions and "cold boot" penalty.
